### ISSUE TITLE
Testing of ion channel models from Podlaski et al

### ISSUE THREAD

**Issue by rgerkin** [scidash/neuronunit#54]
[This paper](https://elifesciences.org/content/6/e22152) has a pipeline for comparing ion channel models to each other and to experimental data.  Can this pipeline be wrapped in or refactored to use NeuronUnit?

**Comment by russelljjarvis** [comment]
Yes I looked through the paper, but I couldn't find any associated github repositories. When I really want to find the code that was used to build a paper, I often google search 'authors name, github'. I might do that later.

**Comment by rgerkin** [comment]
The API is detailed [here](https://icg.neurotheory.ox.ac.uk/api/#!/chs/App_Channel_traces).  We can use this to programmatically get traces used to define tests for ion channels.
